### ISSUE TITLE
3rd party extensions marketplace. 

### ISSUE THREAD

**Issue by rostislav-simonik** [prismagraphql/prisma#2186]
Dear Prisma Team and others, 

We are strongly thinking about to extend prisma framework and implement some scala extensions. Like new queries, new filter functions. Are you planning push given service  in way, that you provide vanilla service interface with basic functionality, but additionally developers can purchase and enable 3rd party extensions, e.g. localised text type or orderByGeoCoordinates filter.

In other words whether are you planning to introduce prisma marketplace for 3rd party plugins and extensions?

If yes then we would like to start discussion how we could participate. 

Thanks for any feedback.
